docs/html/installation.md

```diff
@@ -45,6 +45,34 @@ More details about this script can be found in [pypa/get-pip]'s README.
 
 [pypa/get-pip]: https://github.com/pypa/get-pip
 
+### Standalone zip application
+
+In addition to installing pip in your environment, pip is available as a
+standalone [zip application](https://docs.python.org/3.11/library/zipapp.html).
+This can be downloaded from <https://bootstrap.pypa.io/pip/pip.pyz>. There are
+also zip applications for specific pip versions, named `pip-X.Y.Z.pyz`.
+
+The zip application can be run using any supported version of Python:
+
+```{pip-cli}
+$ python pip.pyz --help
+```
+
+If run directly:
+
+```{pip-cli}
+$ pip.pyz --help
+```
+
+then the currently active Python interpreter will be used.
+
+```{note}
```

Comment:
Suggestion: put the admonition at the top of the section or after the first paragraph, coz people skim 😅

Reply:
lol, good point. Done.